DistributedTransaction/DistributedTransaction.tla

```diff
@@ -120,10 +120,8 @@ TypeOK == /\ req_msgs \in SUBSET ReqMessages
                                           type : {"prewrite_optimistic",
                                                   "prewrite_pessimistic",
                                                   "lock_key"}]]
-          /\ \A k \in KEY :
-              \* At most one lock in key_lock[k]
-              \A l, l2 \in key_lock[k] :
-                l = l2
+          \* At most one lock in key_lock[k]
+          /\ \A k \in KEY: Cardinality(key_lock[k]) = 1

```

Comment:
In case of no lock exists, should the cardinality be `<= 1`?

Reply:
This error has been fixed and has been rebased into one commit.